xml/art_migration.xml

```diff
@@ -404,8 +404,9 @@
     <important>
      <title>Manual Installation Is Recommended</title>
      <para>
-      Although it is possible to use cobbler, we recommend performing a manual
-      installation as described in <xref linkend="book.install"/>.
+      If the server's boot mode is set to Legacy BIOS boot, then it is possible
+      to use cobbler. However, if the boot mode of the server is set to UEFI
+      boot, manual installation is required.
```

Comment:
"recommended" (Titel) oder "required" (Absatz)? Das beißt sich.
Vielleicht kannst du das auch ein bisschen vereinfachen und parallelen Satzbau verwenden: "If the server boots in Legacy BIOS mode, install with Cobbler. However, if the server boots in UEFI mode, install manually."